### DISCUSSION TITLE
No checker found for process

### DISCUSSION THREAD

**Discussion by DrDanielMetz** [eclipse-tractusx/tractusx-edc#868] [Q&A]
I am getting log messages like the below one:
´No checker found for process e0c96684-ec7d-4277-8a13-898d330f42ba. The process will not advance to the COMPLETED state.´

This is already discussed in this bug ticket:

https://github.com/eclipse-tractusx/tractusx-edc/issues/799

But, my questions are:

1. How critical is this bug? Can it lead to a severe problem (e.g., no data transfer is possible after some time)? Or is it just only annoying as the log file has hundreds of those log messages?
2. Is there anything that can be done in the meantime (until a fix is available on tractusx) to stop the unending loops?

**Comment by ndr-brt** [comment]
this is not a bug, it's a deprecated feature that prints an annoying log, it will disappear in the next released version.
the log won't appear in version [0.6.0-rc1](https://github.com/eclipse-tractusx/tractusx-edc/releases/tag/0.6.0-rc1) already

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
